web/src/lib/components/pages/record/record.tsx

```diff
@@ -406,11 +407,15 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
     for (let i = 0; i < SET_COUNT; i++) {
       // For the sentences elements, we need to
       // figure out where each item is positioned.
+      // When rerecording, this is determined by reRecordIndex.
+      // Otherwise, this is determined by the number of sentences recorded.
       let className = 'text-box';
       let length = this.state.recordings.length;
-      if (i < length) {
+      let reRecordIndex = this.state.reRecordIndex;
+      let recordIndex = this.state.isReRecord ? reRecordIndex : length;
```

Comment:
please use `const` (instead of `let`) where possible :)

Reply:
Sure thing. When I fix this, should I stack a new commit on top of this one, or force-push an update to this commit?

Reply:
whichever you prefer, we don't have a strict policy on this :)